README.md

```diff
@@ -115,7 +115,7 @@ esm or cjs:
 - https://unpkg.com/romcal/dist/esm/romcal.js
 - https://unpkg.com/romcal/dist/cjs/romcal.js
 
-Make sure to use a fixed version in production like https://unpkg.com/romcal@3.0.0-dev.1/dist/cjs/romcal.js as passing no version will redirect to the latest version which might contain breaking changes in the future.
+Make sure to use a fixed version in production like https://unpkg.com/romcal@3.0.0-dev.7/dist/cjs/romcal.js as passing no version will redirect to the latest version which might contain breaking changes in the future.
```

Comment:
The version number in the URL could be updated when releasing a new version via GitHub CI. Same below on L146-147.

This is not an issue of this PR though. :wink:

Reply:
Good idea.